client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.ts

```diff
@@ -137,16 +136,14 @@ export const useStocktakeColumns = ({
             if ('lines' in row) {
               const { lines } = row;
               const expiryDate =
-                ArrayUtils.ifTheSameElseDefault(lines, 'expiryDate', null) ??
-                '';
-              return (
-                (expiryDate && Formatter.expiryDate(new Date(expiryDate))) ||
-                t('multiple')
-              );
+                ArrayUtils.ifTheSameElseDefault(
```

Comment:
I've noticed that the method `getStocktakeReasons` is calling a hook, which is naughty!
I would suggest adding something [like this](https://github.com/openmsupply/open-msupply/compare/2429-Change-Formatter.naiveDateTime-to-Formatter.toISOString...2429-Change-Formatter.naiveDateTime-to-Formatter.toISOString(hook-usage))